docker/gatling/Dockerfile

```diff
@@ -1,17 +1,15 @@
 # Dockerfile for building gatling image with benchmark Tests
 FROM denvazh/gatling:2.3.1
 
-# TODO - Discuss if this is needed now.
-# ADD http://abondance.internal.forgerock.com/pkg/clients/gatling/ext/ldap/gatling-ldap-full-0.0.4-SNAPSHOT.zip /
-# RUN mkdir /ldap-plugin
-# RUN unzip /gatling-ldap-full-0.0.4-SNAPSHOT.zip -d /ldap-plugin
-
-#RUN cp /ldap-plugin/* /opt/gatling/lib/
-# Add local files
+# In case of you want to add custom gatling libraries
```

Comment:
In case you want to add custom Gatling libraries,
make sure you you add them to image and copy to the appropriate place.

Reply:
Will fix, thanks